packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -160,4 +187,132 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe (".toBeVisible", () => {
+    context("when the modal is visible", () => {
+      it("returns the assertion instance", () => {
+        const element = render(
+          <Modal testID="id" visible={true} />,
+        );
```

Comment:
Nit: Destructuring can help make the queries simpler 🙂 
```suggestion
        const { getByTestId } = render(
          <Modal testID="id" visible={true} />,
        );
```